website/content/en/docs/building-operators/golang/migration.md

```diff
@@ -392,6 +392,8 @@ Note that:
 - `operator-sdk generate crds` is replaced with `make manifests`, which generates CRDs and RBAC rules.
 - `operator-sdk build` is replaced with `make docker-build IMG=<some-registry>/<project-name>:<tag>`.
 
+`IMAGE_BUILDER` can be used here to specify your container engine. default is `docker`.
```

Comment:
Have we already pulled this in from Kubebuilder in golang, since we are modifying its tutorial here?

Reply:
Truth be told I just did a search and replace on all the docs that mention `build`.
I can remove if it doesn't belong here.